I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 23, in script call
    call fn_dictionary()
  File "game/fn_dictionary.rpy", line 36, in script
    $ r = renpy.random.choice(dict_enemigos.keys)
  File "game/fn_dictionary.rpy", line 36, in <module>
    $ r = renpy.random.choice(dict_enemigos.keys)
TypeError: object of type 'builtin_function_or_method' has no len()

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 23, in script call
    call fn_dictionary()
  File "game/fn_dictionary.rpy", line 36, in script
    $ r = renpy.random.choice(dict_enemigos.keys)
  File "D:\Renpy\Renpy-8\renpy-8.3.4-sdk\renpy\ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Renpy\Renpy-8\renpy-8.3.4-sdk\renpy\python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/fn_dictionary.rpy", line 36, in <module>
    $ r = renpy.random.choice(dict_enemigos.keys)
  File "lib/python3.9/random.py", line 346, in choice
TypeError: object of type 'builtin_function_or_method' has no len()

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.7.25031702
Screen examples 1.0
Tue Apr 15 20:30:27 2025
